src/controller/LoginController.ts

```diff
@@ -17,25 +17,36 @@
 import * as express from 'express'
 import * as urlparse from 'url-parse'
 import {
+    decryptCookie,
     getAuthorizationURL,
+    getCSRFCookieName,
     getTokenEndpointResponse,
     getTempLoginDataCookie,
     getTempLoginDataCookieName,
     getCookiesForTokenResponse,
-    getAuthCookieName, generateRandomString
+    getAuthCookieName,
+    generateRandomString,
 } from '../lib'
 import {config} from '../config'
+import {ValidateRequestOptions} from '../lib/validateRequest';
```

Comment:
This could be exported in `lib/index.ts` and then imported together with other classes/methods from `../lib` (this import could be changed in a few files.)